### ISSUE TITLE
I found a fix for this error 

### ISSUE THREAD

**Issue by envy2456** [MCMrARM/mc-w10-version-launcher#310]
![unknown_2024 08 11-20 20](https://github.com/user-attachments/assets/bb146525-6e8e-4931-aa14-8d7ac1530be2)

just enable the first thing here

![unknown_2024 08 11-20 21](https://github.com/user-attachments/assets/946919fe-5b65-493e-b11c-fb74585e75b0)

**Comment by GreatNateDev** [comment]
Yes this is listed in the [README](https://github.com/MCMrARM/mc-w10-version-launcher?tab=readme-ov-file#prerequisites)

**Comment by envy2456** [comment]
Unfortunately nobody reads these things these days including me
